### PR TITLE
Replace Blood Contract with a Berserk Pack

### PR DESCRIPTION
**Pull request by Linzolle**
## About The Pull Request
Removes blood contract from Bubblegum's drop pool, replaces it with a box full of pills that increase your unarmed damage. You can overdose on the pills to increase your damage even more, but you won't be able to hold any items.

## Why It's Good For The Game
Blood Contract is kinda a shitty and item and this is the first good thing I came up with to replace it with.

## Changelog
:cl:
del: Removed blood contract from bubblegum drop pool
add: demon kit, pop some pills to make your fists extra strong. obtained from bubblegum.
/:cl:

### CODE REVIEW

**Comment by Arturlang** [comment]
Seems pretty useless for miners, their stuff does way more damage when they need it and is ranged.

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -2329,3 +2329,38 @@
...
+	to_chat(H, "<span class='warning'>The adrenaline rushing through your fists fades away.</span>")
+
+/datum/reagent/berserk/overdose_start(mob/living/carbon/human/H)
+	H.dna.species.punchdamagelow = 25
+	H.dna.species.punchdamagehigh = 30
+	H.dna.species.punchstunthreshold = 30
+	to_chat(H, "<span class='userdanger'>You feel power surging through your veins!</span>")
+
+/datum/reagent/berserk/overdose_process(mob/living/L)
+	if (L.get_active_held_item() || L.get_inactive_held_item()) //is there a better way to do this
```
Comment (Putnam3145) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: yes:
```
    if(L.drop_all_held_items())
        to_chat(L, "<span class='warning'>Your fists quiver with energy, unable to get a hold of anything!</span>")

Comment (Linzolle) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm]: ah

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 2]
```diff
@@ -2329,3 +2329,38 @@
...
 	M.adjustArousalLoss(2)
 	..()
+
+/datum/reagent/berserk
+	name = "hell powder"
+	id = "berserk"
+	description = "A drug from Hell that increases the user's muscle strength to great levels."
+	color = "#751717"
+	taste_description = "rage"
+	overdose_threshold = 50
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm:2339]: suggestion made their PH be like 12~ so people dont meme them into beakers to get 50u injectors?

Comment (Linzolle) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm:2339]: that would actually be funny emergent gameplay, bluespace syringe someone and they can't hold items

**Comment by YakumoChen** [comment]
imagine using these with north star gloves or carp

I would have noted that the berserk pack in doom also full healed you but this is actually pretty hilariously strong in the right use cases.

Make it do a red overlay

**Comment by Ghommie** [comment]
The new suggested drop is yet another shade of stimulants, with a quirky "stronger punches" effect that can lead to consistency issues with other, theorical or not, features that also modify the character's raw brawling strength.
Tbh, while I totally understand some people may be salty about the Blood Contract being a grey zone for the current badministration team, this change is too bland to be approvable imo.
We can discuss about alternatives in the chat, but I'm not letting this sit in the limbo. Feel free to bug another maintainer to reopen and merge it.

**Pull request closed without merging**